woocommerce-gateway-stripe.php

```diff
@@ -777,7 +777,7 @@ function() {
 	'before_woocommerce_init',
 	function() {
 		if ( class_exists( '\Automattic\WooCommerce\Utilities\FeaturesUtil' ) ) {
```

Comment:
I just noticed the snippet to enable HPOS is slightly different in the [guide](https://github.com/woocommerce/woocommerce/wiki/High-Performance-Order-Storage-Upgrade-Recipe-Book#declaring-extension-incompatibility).

```suggestion
		if ( class_exists( '\Automattic\WooCommerce\Utilities\FeaturesUtil::class' ) ) {
```

Reply:
I think it's fine as is; the `::class` property is just getting the string representation of the class, yeah? And note that the guide doesn't include the quotes we have. Since we're using quotes I think what we already have is fine. Probably even better?

The string representation returned by `::class` will be wrong if the class doesn't exist, so it makes more sense IMO to not use that. 🤔

Reply:
Yeah, I forgot to remove quotes. Let's leave it as it is now.